### ISSUE TITLE
One set of NN parameters for one element?

### ISSUE THREAD

**Issue by brucefan1983** [brucefan1983/GPUMD#249]
* Currently, all elements share the same set of neural network (NN) parameters in NEP.
* We need to try to use a separate set of NN parameters for each element. Can this improve the accuracy? Will this reduce the efficiency?

**Comment by erhart1** [comment]
With NN parameters you mean those that define the architecture of the NN?

**Comment by brucefan1983** [comment]
Yes, I mean the connection weights and biases. Seems this is one option in N2P2 for multi-component systems.

**Comment by brucefan1983** [comment]
I may misunderstand you. I do not mean the "hyperparameters", but the set of (many) connection weights and biases.

**Comment by erhart1** [comment]
Thanks for the clarification.

**Comment by brucefan1983** [comment]
Doen with PR #277
